circuit_maintenance_parser/parsers/colt.py

```diff
@@ -44,7 +44,7 @@ def parse_subject(self, subject):
         """
         data = {}
         search = re.search(
-            r"\[.+\]\s([A-Za-z\s]+).+?(CRQ\w+-\w+)\s(\d+/\d+/\d+\s\d+:\d+:\d+\s+[A-Z]+).+?(\d+/\d+/\d+\s\d+:\d+:\d+\s+[A-Z]+).+?([A-Z]+)",
+            r"(?:\[.+\]\s)?([A-Za-z\s]+):?\s+?(CRQ\w+-\w+)\s(\d+/\d+/\d+\s\d+:\d+:\d+\s+[A-Z]+).+?(\d+/\d+/\d+\s\d+:\d+:\d+\s+[A-Z]+).+?([A-Z]+)",
```

Comment:
Would you mind adding an example or two of the non `[ EXTERNAL ]` subject lines to the docstrings here and below, just for clarity?